specs/core/0_beacon-chain.md

```diff
@@ -135,29 +164,29 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 | `MIN_VALIDATOR_REGISTRY_CHANGE_INTERVAL` | `2**8` (= 256) | slots | 25.6 minutes |
 | `POW_RECEIPT_ROOT_VOTING_PERIOD` | `2**10` (= 1,024) | slots | ~1.7 hours |
 | `SHARD_PERSISTENT_COMMITTEE_CHANGE_PERIOD` | `2**17` (= 131,072) | slots | ~9 days |
-| `SQRT_E_DROP_TIME` | `2**17` (= 131,072) | slots | ~9 days |
 | `COLLECTIVE_PENALTY_CALCULATION_PERIOD` | `2**20` (= 1,048,576) | slots | ~73 days |
-| `DELETION_PERIOD` | `2**22` (= 16,777,216) | slots | ~290 days |
+| `EMPTY_VALIDATOR_EXPIRY` | `2**22` (= 16,777,216) | slots | ~290 days |
```

Comment:
Would "expiry" sound more like the "timestamp"? What do you think about `EMPTY_VALIDATOR_EXPIRY_PERIOD`?

Reply:
Naming is hard :) This constant refers to the amount of time a validator entry with a balance of zero stays in the registry before being cleaned up. What about `ZERO_BALANCE_VALIDATOR_TTL`?

Reply:
`ZERO_BALANCE_VALIDATOR_TTL` sounds good to me.